static/js/components/GroupSources.jsx

```diff
@@ -0,0 +1,250 @@
+import React, { useEffect, Suspense } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import Grid from "@material-ui/core/Grid";
+// import BuildIcon from "@material-ui/icons/Build";
```

Comment:
Remove these commented out lines here and below

Reply:
done